packages/paste-website/src/pages/components/skeleton-loader/index.mdx

```diff
@@ -0,0 +1,233 @@
+  ButtonsExample,
+  AvatarExample,
+  IconExample,
+  CardExample,
+  TableExample,
+} from '../../../component-examples/SkeletonLoaderExamples';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/skeleton-loader"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/skeleton-loader/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Skeleton Loader"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Skeleton Loader"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/skeleton-loader"
+  storybookUrl="/?path=/story/components-skeleton-loader--default"
+  data={props.data.allPasteComponent.edges}
+  packageStatus={props.data.allAirtable.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{SkeletonLoader}}>{`<SkeletonLoader />`}</LivePreview>
+
+## Guidelines
+
+### About Skeleton Loader
+
+The Skeleton Loader displays a placeholder representing the page's content before the page is ready to display. This component is used to improve [perceived performance](https://developer.mozilla.org/en-US/docs/Learn/Performance/Perceived_performance). This tends to improves user experience by reducing load time frustration and making the page feel more responsive.
+
+This loader is an alternative to the [Spinner component](/components/spinner). Rather than showing an abstract spinner, it frames the content of what is to come which creates anticipation.
+
+The component is designed to be used in place of the content being loaded. For example:
+
+<LivePreview scope={{SkeletonLoader, Text}} noInline>
+  {CompositionExample}
+</LivePreview>
+
+### Accessibility
+
+- The animation can be disabled by enabling the `prefers-reduced-motion` setting at the OS level.
+- Skeleton Loader has the `aria-busy` attribute set to `true` by default. This is to indicate that the content is loading.
+
+## Examples
+
+### Default Skeleton Loader
+
+The default Skeleton Loader can be used to represent a line of text. It uses the `sizeIcon20` token for the `height`, and `borderRadius20` for the `border-radius`. The `width` will be 100% unless a different `width` is specified.
+
+<LivePreview scope={{SkeletonLoader}}>{`<SkeletonLoader />`}</LivePreview>
+
+### Changing the Skeleton Loader height
+
+Use the `height` prop to change the height of the Skeleton Loader.
```

Comment:
Would it be useful to say here that the height could be in px _or_ an icon size? It wasn't clear to me on first glance. This could be solved by adding another example that uses icon size for the height. Is there any other type of value one could put into that prop or just those two?

Reply:
Added some clarifying text.